crates/nu-command/src/path/exists.rs

```diff
@@ -152,4 +170,26 @@ mod tests {
 
         test_examples(SubCommand {})
     }
+
+    #[test]
+    fn test_check_symlink_exists() {
+        use nu_test_support::{nu, playground::Playground};
+
+        let symlink_target = "symlink_target";
+        let symlink = "symlink";
+        Playground::setup("rm_test_symlink", |dirs, sandbox| {
+            #[cfg(not(windows))]
+            std::os::unix::fs::symlink(dirs.test().join(symlink_target), dirs.test().join(symlink))
+                .unwrap();
+            #[cfg(windows)]
+            std::os::windows::fs::symlink_file(
+                dirs.test().join(symlink_target),
+                dirs.test().join(symlink),
+            )
+            .unwrap();
+
+            let shell_res = nu!(cwd: sandbox.cwd(), "'symlink' | path exists -n");
```

Comment:
To test `-n` works, I think you need to remove `symlink` target first.  Then try `'symlink' | path exists -n`

Reply:
Sorry for late reply.

A points to B. (A -> B). `A` is `symlink`, `B` is `symlink target`.

Without the flag -n, `path exists` checks whether the A exists and then B... But what if B does not exist and just A exists? i've added `-n` to cover that.


below line just creates the A, not B. Which is fine. Why would I need to remove the A?

```
std::os::unix::fs::symlink(dirs.test().join(symlink_target), dirs.test().join(symlink)).unwrap();
```

test case checks_if_existing_file_exists() covers that situation i suppose.